requirements.txt

```diff
@@ -1,2 +1,3 @@
 pymodbus3 == 1.0.0
 pyserial == 2.7
+pymodbus
```

Comment:
according to the https://www.python.org/dev/peps/pep-0496/ you can put conditions here
```
pymodbus3 == 1.0.0; python_version >= '3.0'
pymodbus; python_version <= 3.0
```
and pin the specific version of `pymodbus`